en/reference/query-language-reference.html

```diff
@@ -450,6 +450,12 @@ <h2 id="where">where</h2>
             so the fuzzy match will be performed only with the suffix left.
             Default is 0 (i.e. <code>fuzzy</code> will match across whole query)</td>
         </tr>
+        <tr>
+          <td><a href="#prefix">prefix</a></td>
+          <td>If <code>true</code>, a string is considered a match when it's possible to transform a
+            <em>prefix</em> of the candidate string to the query string using at most <code>maxEditDistance</code>
+            edits. Default is <code>false</code>, which means that the entire string is considered.</td>
```

Comment:
Consider adding a link to `text-matching.html#fuzzy-prefix-match` for more details.

Reply:
Done